### ISSUE TITLE
Allow to overwrite only rule options 

### ISSUE THREAD

**Issue by ncoden** [sasstools/sass-lint#601]
I would like to have a `.sass-lint.yml` within only the rules or options I want to overwrite.
I have currently something like :
```yaml
# Linter Options
options:
  merge-default-rules: true

# Rule Configuration
rules:
  property-sort-order: [1, { // the most compact syntax I found
      order: smacss
    }]
```

It would be great if you allow:
```yaml
...
rules:
  property-sort-order:
    order: smacss
```

**Comment by BPScott** [comment]
Having two ways to parse and understand a config file, where code would have to check if the value of a rule is an number, object or array and update a single item accordingly would be complicated and has the potential for bugs as array/object merging is actually trickier than it first appears as there are often many ways of resolving a merge and it is never clear if you want to merge or overwrite content.

```
# Linter Options
options:
  merge-default-rules: true

# Rule Configuration
rules:
  property-sort-order: 
    -1
    - order: smacss
```

Would be the 'usual' yaml way of writing out the config file of an array that contains an object. Considering the difference is between this and your prefered option is one extra line it when you define the errorlevel of the rule it doesn't seem worth the complication.

**Comment by DanPurdy** [comment]
I'm going to close this as I dont think it really follows the direction we would like to take and I'm not sure we would ever consider allowing this sort of overriding when we want everyone to be declarative in the levels of errors they expect for a rule etc. If there's significant interest then we may change our opinion here but for now I believe this is a wontfix. Sorry.
